connectors/src/connectors/intercom/temporal/activities.ts

```diff
@@ -70,9 +73,20 @@ export async function syncHelpCentersActivity({
     dataSourceName: dataSourceConfig.dataSourceName,
   };
 
-  console.log("syncHelpCentersActivity", {
-    connectorId,
-    dataSourceConfig,
-    loggerArgs,
+  const intercomClient = await getIntercomClient(connector.connectionId);
+  const helpCentersOnDb = await IntercomHelpCenter.findAll({
+    where: {
+      connectorId,
+    },
+  });
+
+  helpCentersOnDb.map(async (helpCenter) => {
+    await syncHelpCenter({
```

Comment:
Each of these call should likely be an activity? 

What we're doing in all other connectors is have the "Ids" float through the workflow.

One activity pulls the IDs
the workflow schedules the activities for each thing to sync

That way on activity failure we retry directly the one that failed.

Reply:
I did not bother with that because I'm expecting most companies to have only 1 help center & because I've been advised to put as little code as possible in the workflow, but happy to change that  👍🏻 

I'll also make some light changes to prepare for the sync of Conversations & use the Signal feature  to notify when we need to launch this workflow immediately after a change in permissions. 

I'll ping on Slack when ready for a new check

Reply:
:raised_hands: